### PR TITLE
Show latest Stylelint version in README

### PR DESCRIPTION
**Pull request by ybiquitous**
> Which issue, if any, is this issue related to?

None.

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ To see test results, click the status badge for the relevant package in the list
...
+| Package                                                                                                                                                                                                                     |                                                                                                                                                                                                                                                                                                               Latest Stylelint (16.7.0) |                                                                                                                                                                                                                                                                                                                    Next Stylelint |
```
Comment (romainmenke) [inline README.md:13]: Should we maybe als use an npm badge here?
So that it is always up to date?

Comment (ybiquitous) [reply README.md:13]: Ah, nice idea! I'll update it to use badge 👍🏼

Comment (ybiquitous) [reply README.md:13]: There's a slight shift between the image and the text, but do you have any idea?

image

Comment (ybiquitous) [reply README.md]: I found the `label` query option:

```markdown
[![Latest Stylelint](https://img.shields.io/npm/v/stylelint.svg?label=Latest+Stylelint)](https://www.npmjs.com/package/stylelint)
```

How about this?

[Latest Stylelint](https://www.npmjs.com/package/stylelint)

Comment (Mouvedia) [reply README.md]: > There's a slight shift between the image and the text, but do you have any idea?
> 
> image

Try adding `align="middle"`.

Comment (ybiquitous) [reply README.md]: Isn't `align` deprecated?

Comment (Mouvedia) [reply README.md]: All browsers support it and `style="…"` would be ignored by GitHub so you don't have much choice.

Comment (ybiquitous) [reply README.md]: How do we set `align` in Markdown?

Comment (Mouvedia) [reply README.md]:
```html
<a href="https://www.npmjs.com/package/stylelint">Latest Stylelint</a> <img align="top" alt="Latest Stylelint" src="https://img.shields.io/npm/v/stylelint.svg" />
```

<a href="https://www.npmjs.com/package/stylelint">Latest Stylelint</a> Latest Stylelint

Comment (romainmenke) [reply README.md]: Markdown also seems ok to me :)
We can improve as we go along.

**Review by romainmenke — APPROVED** [review]
LGTM 🚀

**Pull request merged**